src/privileged/shieldUtils/jsm/StudyUtilsBootstrap.jsm

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+var EXPORTED_SYMBOLS = ["Bootstrap"];
+
+const { utils: Cu } = Components;
+Cu.import("resource://gre/modules/Services.jsm");
+
+this.Bootstrap = function(config, studyUtils) {
+  return {
+    /**
+     * Use console as our logger until there is a log() method in studyUtils that we can rely on
+     */
+    log: console,
+
+    /**
+     *
+     * @param manifest
+     * @param reason
+     * @returns {Promise<void>}
```

Comment:
We should see if we have JSDoc checking set up in our .eslintrc config file. These `@param` tags look old/fake.
But that should be a separate issue/PR.

Reply:
Your suggestion in #61 is now included in this PR :)